components/monitoring/logging/staging/monitoring-logging-splunk-forwarder.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: external-secrets.io/v1beta1
```

Comment:
the file is just adding a external secret, Id expect the name to indicate that.

Reply:
I just kept the same filename pattern similar to
`/components/monitoring/logging/base/external-secrets/monitoring-logging-cloudwatch-forwarder.yaml`
we can refactor both of those names later on. Lets first find out if the Splunk forwarder works at all...

Reply:
Well, that other fil is in an "external-secrets" directory - this one isn't.

Reply:
Fixed.